packages/suite/src/views/wallet/trading/common/TradingSelectedOffer/TradingInfo/TradingInfoItem.tsx

```diff
@@ -59,11 +63,17 @@ export const TradingInfoItem = ({
                     <Text variant="tertiary" typographyStyle="hint" as="div">
                         <Row>
                             {accountLabelPrefix}&nbsp;
-                            <AccountLabel
-                                account={account}
-                                showAccountTypeBadge
-                                accountTypeBadgeSize="small"
-                            />
+                            {type === 'exchange' && receiveAddress ? (
+                                <Tooltip content={receiveAddress} hasArrow>
+                                    <StyledReceiveAddress>{receiveAddress}</StyledReceiveAddress>
+                                </Tooltip>
```

Comment:
```suggestion
                              <TokenAddressRow
                                    tokenContractAddress={receiveAddress}
                                    shouldAllowCopy={true}
                                    onCopy={() => copyAddressToClipboard(receiveAddress)}
                                    typographyStyle="hint"
                                    variant="tertiary"
                                />
```

Reply:
sync with @TomasBoda, he is going to use this component too, maybe he will rename it/modify it

Reply:
Nice, that's better component for it, thanks. Alright, we'll sync.